README.rst

```diff
@@ -48,10 +48,10 @@ index page is `here <http://pypi.python.org/pypi/pyFFTW>`_.
 
 Requirements (i.e. what it was designed for)
 --------------------------------------------
-- Python 2.7 or greater (Python 3 is supported)
-- Numpy 1.6
+- Python 2.7 or greater (Python 3.4+ is also supported)
+- Numpy 1.10
```

Comment:
It's probably worth keeping [this]( https://github.com/conda-forge/pyfftw-feedstock/blob/ad4b44b1bc636618c457ed3cb38d15fd4e046640/recipe/meta.yaml#L24-L26 ) in mind. We can obviously bump what we are building against. So this doesn't necessarily constitute a problem. Just depends on how much compatibility we are comfortable keeping.

Reply:
I would be fine with 1.8 as well.  There is no problem running with 1.6 at the moment, it is just that at some point it would be nice to be able to drop some of the workarounds for older numpy.  Perhaps use 1.8 for the next release and bump again to 1.10 at some future date?

Reply:
Sure 1.8 seems fine.

Certainly not opposed to bumping more. It's worth noting that 1.10 wouldn't likely get lots of support in c-f, but 1.11 would. So it might make more sense to go to 1.11 or at least build with it in c-f. Assuming we want to go that way.

Reply:
As a counter to my original point, would add that Dask is tested with NumPy 1.10.4 at the lower end. Based on the `cbrt` wrapping issue seen here, we know it is not compatible with NumPy 1.9. So that might be another argument to move to NumPy 1.10 at a minimum. However it is an optional dependency.